src/server-code/admin-server.js

```diff
@@ -189,13 +189,14 @@ export default function runAdminServer(serverFalcorModel) {
       });
     }
   });
-
+  app.get('/robots.txt', (req, res) => {
```

Comment:
Just for readability if you could just add a newline above and below the `app.get` body